src/pybind/mgr/dashboard/controllers/motd.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+
+import json
+
+from ceph.utils import datetime_now, str_to_datetime
+
+from .. import mgr
+from ..plugins.motd import MotdData
+from . import ApiController, ControllerDoc, EndpointDoc, RESTController
+
+MOTD_SCHEMA = {
+    "message": (str, "The message."),
+    "severity": (str, "The severity, e.g. 'info', 'warning' or 'danger'."),
+    "expires": (str, "The expiration date in ISO 8601. Does not expire if empty.")
+}
+
+
+@ApiController('/motd')
+@ControllerDoc("Display message of the day (MOTD)", "Dashboard")
+class MessageOfTheDay(RESTController):
+
+    @EndpointDoc("Get the message of the day",
+                 responses={200: MOTD_SCHEMA})
+    def list(self):
+        value: str = mgr.get_module_option('motd')
+        if not value:
+            return None
+        data: MotdData = MotdData(**json.loads(value))
+        # Check if the MOTD has been expired.
+        if data.expires:
+            expires = str_to_datetime(data.expires)
+            if expires < datetime_now():
+                return None
+        return data._asdict()
```

Comment:
I've mixed feelings with relocating the ocde because the main entry point for controllers is the `controllers/` directory. But implementing the code in the plugin also has its charm. So everything is together that belongs together.

Reply:
Done

Reply:
Thanks!